packages/core/src/dynamoDb/vue/tableView.ts

```diff
@@ -0,0 +1,79 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+import { ExtContext } from '../../shared'
+import { VueWebview } from '../../webviews/main'
+import { getLogger, Logger } from '../../shared/logger'
+import { telemetry } from '../../shared/telemetry/telemetry'
+import { DynamoDbTableNode } from '../explorer/dynamoDbTableNode'
+import { getTableContent, RowData, TableData } from '../utils/dynamodbUtils'
+import { Key, ScanInput } from 'aws-sdk/clients/dynamodb'
+
+const localize = nls.loadMessageBundle()
+
+interface DynamoDbTableData {
+    TableName: string
```

Comment:
can we make all the keys of this lowercase instead of some upper and some lower

Reply:
Updated